apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1574,30 +1565,21 @@ public static Bundle addResourcesToBundle(Bundle bundle, List<IBaseResource> res
       }
     }
 
-    logBeneIdToMdc(beneIds);
+    logBeneIdToMdc(beneIds.stream().map(Long::valueOf).toArray(Long[]::new));
```

Comment:
can we avoid some of this by changing the underlying Set<String> to Set<Long>

Reply:
Cleaned it up